### PR TITLE
added docs to make it more clear on how to edit the statusline

### PR DESCRIPTION
**Pull request by theTechGoose**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,109 @@ Lightweight &amp; performant ui plugin for nvchad providing the following :
...
+The config for the status line is in `.config/nvim/lua/core/default_config.lua`
+
+here you will find a line that looks like this:
+```lua
+  statusline = {
+    theme = "default", -- default/vscode/vscode_colored/minimal
+    -- default/round/block/arrow separators work only for default statusline theme
+    -- round and block will work for minimal theme only
+    separator_style = "default",
+    overriden_modules = nil,
```
Comment (lucario387) [inline README.md]: remove this

File: README.md [hunk 2]
```diff
@@ -12,6 +12,109 @@ Lightweight &amp; performant ui plugin for nvchad providing the following :
...
+    -- round and block will work for minimal theme only
+    separator_style = "default",
+    overriden_modules = function(modules)
+          modules[2] = 'hello world'
+    end
+  },
+```
+
+Here is a full example that creates a variable named vim.g.file_path_length which allows you to make your file path in the status line as short or as long as you want.
+```lua
```
Comment (lucario387) [inline README.md:39]: too long, some shorter examples will be preferred

**Comment by siduck** [comment]
https://nvchad.com/docs/config/nvchad_ui is enough. Adding docs here could lead to confusion as docs are already linked above and are concise :nerd_face:

**Comment by theTechGoose** [comment]
Okay, I'm just trying to help as I spent a few days trying to figure this out and was hoping to help others save the time. The only counter-point I would make is that you pointed out that you had another person submit a pull request similar to the two I have submitted today. If there are 2 people that are confused on how to do this do you think there might be a few more?

Just a thought. Please don't kill me 🤓

**Comment by siduck** [comment]
i meant the other guy made PR before and it was closed due to the same reasons yours is closed!
 
if others read the docs then it'd be easy for them to override statusline too :)

**Comment by lucario387** [comment]
I'd argue docs here help. It can also be integrated with nvchad_types for better support. And no, figuring out which module goes with which spot is hard.

**Pull request closed without merging**